api/handlers/clients/get_test.go

```diff
@@ -1,17 +1,29 @@
 package clients_test
 
 import (
+	"database/sql"
 	"net/http"
 	"testing"
 
-	handler "github.com/fanfit/login/api/handlers"
-	"github.com/fanfit/login/api/handlers/clients"
-	"github.com/fanfit/login/models/clients/service"
+	"github.com/fanfit/login/models/clients/repository"
 	"github.com/fanfit/login/testutil"
-	"github.com/gin-contrib/cors"
-	"github.com/gin-gonic/gin"
 )
 
+var testUser = repository.GetClientByEmailRow{
+	ID:             0,
```

Comment:
The test user should have populated fields that can be differentiated from a default empty object. Otherwise we risk running into problems like yesterday